tests/integration/vlan_test.py

```diff
@@ -90,9 +91,39 @@ def test_add_and_remove_two_vlans_on_same_iface(eth1_up):
     assert not current_state[Interface.KEY]
 
 
+def test_two_vlans_on_eth1_change_mtu(eth1_up):
+    desired_state = statelib.show_only(('eth1',))
+    vlans_state = copy.deepcopy(TWO_VLANS_STATE)
+    desired_state[Interface.KEY].extend(vlans_state[Interface.KEY])
+    for iface in desired_state[Interface.KEY]:
+        iface[Interface.MTU] = 2000
+    libnmstate.apply(desired_state)
+
+    for iface in desired_state[Interface.KEY]:
+        if iface[Interface.NAME] != 'eth1.101':
+            iface[Interface.MTU] = 2200
+    libnmstate.apply(desired_state)
+
+    eth1_vlan_iface_cstate = statelib.show_only(('eth1.101',))
+    assert eth1_vlan_iface_cstate[Interface.KEY][0][Interface.MTU] == 2000
+
+
+def test_two_vlans_on_eth1_change_mtu_using_commit(eth1_up):
```

Comment:
The test name seems to imply that commit is used but actually it is not.

Reply:
Yes, I've changed the name to `test_two_vlans_on_eth1_change_mtu_rollback`.